### PR TITLE
Use in-container env vars for liveness/readiness probes

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@bparees @smarterclayton question here, because this works for me (I tested mongodb), but the question is, will the env var be substituted by the container env if I call this this way?

File: examples/db-templates/mongodb-ephemeral-template.json [hunk 1]
```diff
@@ -125,10 +125,10 @@
...
                 ],
                 "resources": {
-		    "limits": {
-			"memory": "${MEMORY_LIMIT}"
-		    }
-		},
+                  "limits": {
+                    "memory": "${MEMORY_LIMIT}"
+                  }
+                },
```
Comment (bparees) [inline examples/db-templates/mongodb-ephemeral-template.json]: undo these spacing changes.

**Comment by bparees** [comment]
> @bparees @smarterclayton question here, because this works for me (I tested mongodb), but the question is, will the env var be substituted by the container env if I call this this way?

if you tested it, don't you know the answer to that question?  I would have expected that for purposes of exec'ing, the image env would be present, but the container env (as defined in the pod template) would not be.

**Comment by smarterclayton** [comment]
The container env passed on container start is available during exec sessions.

**Comment by mfojtik** [comment]
@bparees fixed spacing.

right, just wanted a sanity check.

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**